test/integration_test/kubevirt_hyperconv_test.go

```diff
@@ -797,14 +839,12 @@ func verifyHotPlugDisk(t *testing.T, testState *kubevirtTestState, hpDisk *hotPl
 		Dash.VerifyFatal(t, isHotplugDiskCollocated(testState, hpDisk), true, fmt.Sprintf("%s was collocated", hpDisk))
 		return
 	}
-	// set the cluster option to reduce the wait time
-	setFixVPSJobFrequency(t, testState.allNodes)
 	require.Eventuallyf(t, func() bool {
 		return isHotplugDiskCollocated(testState, hpDisk)
 	}, time.Hour, 5*time.Second, "%s was not collocated", hpDisk)
 }
 
-func setFixVPSJobFrequency(t *testing.T, allNodes map[string]node.Node) {
+func setFixVPSJobFrequency(t *testing.T, allNodes map[string]node.Node, frequencyInMinutes int) {
```

Comment:
Do we need this function since all it's doing is calling `updateFixVPSJobFrequency` ?

Reply:
> Do we need this function since all it's doing is calling `updateFixVPSJobFrequency` ?

It is just to preserve the commented out code which should really work. At some point, we need to figure out why `volumeDriver.SetClusterOpts` is broken, fix it and use the commented out code.